### PR TITLE
Concf 459 - style collection view

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
ping @Warkot @rogatty

File: extensions/wikia/CollectionView/styles/CollectionView.scss [hunk 1]
```diff
@@ -1,62 +1,28 @@
-@import 'skins/shared/color';
-@import "skins/shared/mixins/columns";
+$miniature-size: 100px;
 
-$infobox-width: 270px; // temporary value used to mach with Khal Drogo Infobox
-$infobox-margin: 15px;
-$infobox-item-margin: 10px;
-$infobox-bg-color-mix-value: 90%;
-$infobox-background: mix($color-page, #000, $infobox-bg-color-mix-value);
+.item-type-item {
```
Comment (rogatty) [inline extensions/wikia/CollectionView/styles/CollectionView.scss]: Isn't it too general? IMO we should have some indication it's collection view.

Comment (Warkot) [inline extensions/wikia/CollectionView/styles/CollectionView.scss]: :+1:

File: extensions/wikia/CollectionView/styles/CollectionView.scss [hunk 2]
```diff
@@ -1,62 +1,28 @@
-@import 'skins/shared/color';
-@import "skins/shared/mixins/columns";
+$miniature-size: 100px;
```
Comment (rogatty) [inline extensions/wikia/CollectionView/styles/CollectionView.scss]: How about `$collection-view-image-size`?

File: extensions/wikia/CollectionView/scripts/CollectionView.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 (function($, vignette) {
-	var images = $('.collection-view-item-image');
+	var images = $('.item-type-image');
```
Comment (rogatty) [inline extensions/wikia/CollectionView/scripts/CollectionView.js]: Why so general? Won't it interfere with infoboxes?

File: extensions/wikia/CollectionView/styles/CollectionView.scss (hunk 2 above)
Comment (bognix) [inline extensions/wikia/CollectionView/styles/CollectionView.scss]: by item I understand image + caption so let's make it $collection-view-image-size

**Comment by rogatty** [comment]
LGTM

**Pull request merged**